### PR TITLE
Make operator admin of created namespaces

### PR DESCRIPTION
**Pull request by philbrookes**
- make operator have admin role in any namespaces it creates 
- add labels to everything we create

### CODE REVIEW

File: pkg/controller/installation/products/rhsso/reconciler.go [hunk 1]
```diff
@@ -101,8 +105,32 @@ func (r *Reconciler) handleAwaitingNSPhase() (v1alpha1.StatusPhase, error) {
...
+			RoleRef: v12.RoleRef{
+				Name:     "admin",
+				Kind:     "ClusterRole",
+				APIGroup: "rbac.authorization.k8s.io",
+			},
+			Subjects: []v12.Subject{
+				{
+					Kind:      "ServiceAccount",
+					Name:      "integreatly-operator",
+					Namespace: "005-test",
```
Comment (aidenkeating) [inline pkg/controller/installation/products/rhsso/reconciler.go]: Should this be a var of some kind? Same above.

File: pkg/controller/installation/products/amqstreams/reconciler.go [hunk 2]
```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/integr8ly/integreatly-operator/pkg/controller/installation/marketplace"
 	"github.com/integr8ly/integreatly-operator/pkg/controller/installation/products/config"
 	coreosv1alpha1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	v12 "k8s.io/api/rbac/v1"
```
Comment (matskiv) [inline pkg/controller/installation/products/amqstreams/reconciler.go]: IMHO `rbacv1` name would be better

File: pkg/controller/installation/products/rhsso/reconciler.go [hunk 3]
```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/integr8ly/integreatly-operator/pkg/controller/installation/marketplace"
 	"github.com/integr8ly/integreatly-operator/pkg/controller/installation/products/config"
 	coreosv1alpha1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	v12 "k8s.io/api/rbac/v1"
```
Comment (matskiv) [inline pkg/controller/installation/products/rhsso/reconciler.go]: Same here, IMHO `rbacv1` name would be better

File: pkg/controller/installation/products/rhsso/reconciler.go (hunk 1 above)
Comment (philbrookes) [reply pkg/controller/installation/products/rhsso/reconciler.go]: Whoops! Yes it should

File: pkg/controller/installation/products/amqstreams/reconciler.go (hunk 2 above)
Comment (philbrookes) [reply pkg/controller/installation/products/amqstreams/reconciler.go]: sure, I'll rename it

File: pkg/controller/installation/marketplace/manager.go [hunk 4]
```diff
@@ -57,6 +57,9 @@ func (m *MarketplaceManager) CreateSubscription(os marketplacev1.OperatorSource,
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: ns,
 			Name:      pkg,
+			Labels: map[string]string{
+				"integreatly": "yes",
```
Comment (matskiv) [inline pkg/controller/installation/marketplace/manager.go]: We already use `integreatly-middleware-service=true` label in 3.x installation:
https://github.com/integr8ly/installation/search?q=integreatly-middleware-service&unscoped_q=integreatly-middleware-service
Maybe we should stick to that?

**Pull request closed without merging**